src/client/src/apps/MainRoute/widgets/analytics/epics/epics.test.ts

```diff
@@ -1,12 +1,13 @@
 import { MockScheduler } from 'rt-testing'
 import { ReferenceActions, ConnectionActions } from 'rt-actions'
 import { analyticsServiceEpic } from './epics'
-import { ActionsObservable } from 'redux-observable'
+import { ActionsObservable, StateObservable } from 'redux-observable'
 import { AnalyticsActions } from '../actions'
 import { Action } from 'redux'
 import { of, Observable } from 'rxjs'
 import { map } from 'rxjs/operators'
 import { ServiceStubWithLoadBalancer } from 'rt-system'
+import { GlobalState } from '../../../../../StoreTypes';
```

Comment:
:(

Reply:
:( :(

Reply:
I thought BaseUrl is supported in CRA

Reply:
`StoreTypes` is at the root of the project, not in any of the subfolders. I hope all this goes away when we switch to multiple packages in the future